tests/HypothesisClient/Credentials/UserManagementCredentialTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace tests\eLife\HypothesisClient\Credentials;
+
+use eLife\HypothesisClient\Credentials\UserManagementCredential;
+use PHPUnit_Framework_TestCase;
+use Serializable;
+
+/**
+ * @covers \eLife\HypothesisClient\Credentials\UserManagementCredential
+ */
+class UserManagementCredentialTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @test
+     */
+    public function it_has_getters()
```

Comment:
That's rather meaningless.

Reply:
I was just moving an existing test, I will review.